lib/chef/provider/package/powershell.rb

```diff
@@ -115,7 +118,8 @@ def build_powershell_package_command(command, version = nil)
           command = [command] unless command.is_a?(Array)
           cmdlet_name = command.first
           command.unshift("(")
-          %w{-Force -ForceBootstrap}.each do |arg|
+          # -WarningAction SilentlyContinue is used to suppress the warnings from stdout
```

Comment:
I guess I have a question as to why you would want to suppress warnings from stdout? 

The other thing is you might want `-erroraction stop` which will blow up powershell if the command errors (or at least throw the error so it can actually be caught ...)

Reply:
Hi @Xorima , I  have used WarningAction because while storing the version from stdout, it also stores with the warnings e.g.
`WARNING: MSG:UnableToDownload �https://go.microsoft.com/fwlink/?LinkID=627338&clcid=0x409� ��
WARNING: Unable to download the list of available providers. Check your internet connection.
1.8.6`
This whole value is stored in the candidate_version as string.